packages/tools/kolibri-cli/src/info/index.ts

```diff
@@ -0,0 +1,92 @@
+import type { Command } from 'commander';
+import os from 'os';
+import { execSync } from 'child_process';
+import path from 'path';
+import fs from 'fs';
+import { PackageJson } from '../types';
+
+// Function to get the binary version
+const getBinaryVersion = (command: string): string => {
+	try {
+		return execSync(`${command} --version`, { encoding: 'utf8' }).trim();
+	} catch {
+		return 'N/A';
+	}
+};
+
+type OSInfo = { platform: string; arch: string; version: string };
+
+// Get operating system information
+const getOsInfo = (): OSInfo => ({
+	platform: os.platform(),
+	arch: os.arch(),
+	version: os.release(),
+});
+
+type Binaries = { node: string; npm: string; yarn: string; pnpm: string };
+
+// Get binaries information
+const getBinariesInfo = (): Binaries => ({
+	node: getBinaryVersion('node'),
+	npm: getBinaryVersion('npm'),
+	yarn: getBinaryVersion('yarn'),
+	pnpm: getBinaryVersion('pnpm'),
+});
+
+type PackageInfo = { [key: string]: string };
+
+// Get relevant packages information
+const getRelevantPackagesInfo = (): PackageInfo => {
+	const packageJsonPath = path.join(process.cwd(), 'package.json');
+	const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8')) as PackageJson;
+
+	const relevantPackages = [
```

Comment:
Vorschlag, damit wir es in Zukunft nicht so oft anpassen müssen: Können wir alles mit `@public-ui/*` matchen?